### PR TITLE
Addded Careers App

### LINKED ISSUE #82: Create app for Careers
View to display the careers page at `/careers/`

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Closes #82 
Deployed: http://52.14.46.160:8000/
username: test
password: test

### CODE REVIEW

**Comment by Monal5031** [comment]
@aashutoshrathi Remove these files:
wsgi/odyssy/careers/templates/careers/__init__.py
wsgi/odyssy/careers/admin.py
Also, please deploy.

File: wsgi/odyssy/careers/templates/careers/careers.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{% endblock %}
+{% block body %}
+        <!-- Content wrapper -->
+        <div class="contents-wrapper">
+            <!-- Contents starts here -->
+            <div class="content career">
+                <h1>Careers @ IIIT Vadodara</h1>
+                <br>
+                <ul class="list-group">
+                    <!--<li class="list-group-item">
```
Comment (Monal5031) [inline wsgi/odyssy/careers/templates/careers/careers.html]: You can remove this commented part.

File: wsgi/odyssy/careers/templates/careers/careers.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                   </li>-->
+		   <li class="list-group-item">
+                        <h4>Faculty</h4> Presently the Indian Institute of Information Technology, Vadodara is being mentored by <a href="http://daiict.ac.in" target="_blank">DAIICT</a>, Gandhinagar. Being the academic mentor, faculty recruitment for IIITV is also managed by DAIICT. The institute invites applications from motivated candidates with research expertise in all areas of
+                        <b>Computer Science and Information Technology</b>. Suitable candidates must have a doctorate in a relevant discipline. We have openings at the levels of Assistant Professor, Associate Professor, and Professor. Visiting faculty positions are also available for short or long term visits. To apply, please submit <b>a curriculum vitae, a publication list, a one-page summary of research
+                    plans, a statement of teaching interests, and the names of three references</b> (with contact information) electronically to <img src="http://iiitvadodara.ac.in/img/email/facultysearch.jpg" style="max-width: 230px;">
```
Comment (Monal5031) [inline wsgi/odyssy/careers/templates/careers/careers.html]: Add this image to static files in basic app and link here.

**Comment by geekyJock8** [comment]
Is there any reason for using "facultysearch.jpg"?
If it's not, replace it with the text 😅.

**Comment by aashutoshrathi** [comment]
@geekyJock8 I'm not sure about it, I felt the same but we are using images for all emails, instead of text.
@singhpratyush Does it have something to do with Email Privacy, or so that search engines can't crawl emails and make it easily accessible or we can change it to text?

**Comment by Monal5031** [comment]
@geekyJock8 @aashutoshrathi I think we should let it be as it is.

**Comment by Monal5031** [comment]
@geekyJock8 @aashutoshrathi let's change it to bold text, it'd be better imo, also easy to copy if needed.

**Pull request merged**